app/controllers/admin/goal_results_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Admin::GoalResultsController < Admin::BaseController
+  skip_before_action :verify_authenticity_token
+  before_action :load_match, :check_status_match, only: :create_all
+
+  def create_all
```

Comment:
```suggestion
  def create
```
thêm all làm j để bị comment thế bé